apispec/insolvency-delta-spec.yml

```diff
@@ -0,0 +1,158 @@
+post:
+  summary: Accepts an incoming insolvency delta, transforms it into an avro schema and puts it onto a Kafka topic.
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/Insolvency'
+    required: true
+  responses:
+    "200":
+      description: Successfully produced message onto Kafka topic.
+    "400":
+      description: Bad request body - validation errors.
+    "401":
+      description: Unauthorised - missing api key in header.
+    "500":
+      description: Internal server error has occured.
+components:
+  schemas:
+    Insolvency:
+      required:
+      - company_number
+      - case_numbers
+      type: object
+      properties:
+        title:
+          type: string
+          enum: ['insolvency']
+        company_number:
+          maxLength: 10
+          type: string
+        case_numbers:
+          type: array
+          items:
+            $ref: '#/components/schemas/Case-number'
+    Case-number:
```

Comment:
Can you add new lines between each component so that it is a bit more readable?

Reply:
Done